include/hermite_multidimensional.hpp

```diff
@@ -215,17 +217,12 @@ inline std::vector<T> hermite_multidimensional_cpp(std::vector<T> &R_mat, std::v
                 prevJump[ii] = 1;
                 std::transform(jumpFrom.begin(), jumpFrom.end(), prevJump.begin(), tmpjump.begin(), std::minus<int>());
                 ullint prevCoordinate = vec2index(tmpjump, resolution);
-                H[nextCoordinate] = H[nextCoordinate] - (static_cast<T>(jumpFrom[ii]-1))*static_cast<T>(R(k,ii))*H[prevCoordinate];
+                H[nextCoordinate] = H[nextCoordinate] - (intsqrt[jumpFrom[ii]-1]/intsqrt[nextPos[k]-1])*static_cast<T>(R(k,ii))*H[prevCoordinate];
 
             }
         }
 
     }
-
-    if (renorm) {
-        H = renormalization(H, dim, resolution);
-    }
-
     return H;
```

Comment:
ah, so the old functions (such as `sqrtfactorial`) are included inline inside `renorm_hermite_multidimensional_cpp`?

By the way, I'm not sure we need to have `_cpp` at the end of these functions, none of the other C++ functions have this?

Reply:
Yes, all that renormalizing is done on the fly...
And yes, it is only this function that have cpp...